tcms/testplans/static/testplans/js/get.js

```diff
@@ -0,0 +1,140 @@
+const TestPlanModule = (function() {
+    'use strict';
+
+    var testplan_id = null,
```

Comment:
I think that it's best to use JS naming convertions (`testPlanId`, `testCases`, etc.), because once we introduce ESLint or other kind of linting it will start complaining.

Reply:
I agree.